lib/orwell/github/user.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Orwell.Github.User do
```

Comment:
I'm using `Orwell.GitHub` instead of `Github`. One of us should update it for consistency. Do you have a preference?